.readthedocs.yaml

```diff
@@ -35,3 +35,5 @@ formats:
 python:
   install:
     - requirements: requirements-doc.txt
+    - method: pip
```

Comment:
What is the default method if pip is not specified?

Reply:
oh it looks like the default is `pip`, so this is superfluous